starfish/test/full_pipelines/api/test_iss_api.py

```diff
@@ -119,5 +125,11 @@ def test_iss_pipeline_cropped_data():
     lab = TargetAssignment.Label()
     assigned = lab.run(label_image, decoded)
 
+    pipeline_log = assigned.attrs[LOG]
+
+    assert pipeline_log[0]['method'] == 'WhiteTophat'
+    assert pipeline_log[1]['method'] == 'FourierShiftRegistration'
+    assert pipeline_log[2]['method'] == 'BlobDetector'
```

Comment:
This is a clever way to test. 👍 

There are substrates here to evaluate the missing components I highlighted above.